HW2/P3/P3.txt

```diff
@@ -0,0 +1,24 @@
+Performance for pure multithreading:
+
+num_threads = 1
+1074.677197 Million Complex FMAs in 5.38133406639 seconds, 199.704605539 million Complex FMAs / second
+
+num_threads = 2
+1074.677197 Million Complex FMAs in 2.83916711807 seconds, 378.518471195 million Complex FMAs / second
+
+num_threads = 4
+1074.677197 Million Complex FMAs in 1.63223195076 seconds, 658.409606857 million Complex FMAs / second
+
+Performance with AVX and multithreading:
```

Comment:
I also got that AVX was slower. I remember Ray mentioning that some machines have to simulate AVX, and I figured that this was the reason for this counterintuitive result.